parsec/arena.c

```diff
@@ -176,6 +176,62 @@ parsec_arena_release_chunk(parsec_arena_t* arena,
     arena->data_free(chunk);
 }
 
+int  parsec_arena_allocate_copy_device_private(parsec_arena_t *arena,
```

Comment:
I do not understand that function? This is intended to allocate memory on the device_private, yet we allocate a single arena_chunk from the `arena->data_malloc`, then we fill-it from the CPU. Either 1) this is allocating CPU memory or 2) this doesn't work with UVM; 3) this cannot and should not be used to allocate GPU device memory?